template.yaml

```diff
@@ -385,11 +372,8 @@ Resources:
       Handler: no.unit.bibs.contents.GetContentFileApiHandler::handleRequest
       Runtime: java11
       MemorySize: 1024
-      AutoPublishAlias: live
       DeploymentPreference:
         Type: AllAtOnce # Or Canary10Percent5Minutes, Linear10PercentEvery1Minute, ...
```

Comment:
You must specify an AutoPublishAlias in your AWS::Serverless::Function to use a DeploymentPreference object, otherwise an error will result.